docs/timelines/timeline-options.md

```diff
@@ -0,0 +1,102 @@
+---
+id: timeline-options
+title: Timeline Options
+sidebar_label: Timeline options
+---
+
+Timelines have a number of options which set how they operate. 
+Press \<Options\> (or the \[Options\] softkey on the top level menu) then the **select** button for the timeline you wish to edit.
+
+--- 
+
+## Handle Tab
+
+All options are the same as for a cue, see [Options](../cues/playback-options.md#handle-tab).
+
+--- 
+
+## Times Tab
+
+![Timeline Options: Times](/docs/images/Timeline-Options-Times.png)
+
+These options let you set the end time and start time of the timeline. These times set the loop points when you are using the 
```

Comment:
End time has been replaced by duration in beta 105

Reply:
Changed